astrocut/asdf_cutouts.py

```diff
@@ -0,0 +1,57 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""This module implements cutout functionality similar to fitscut, but for the ASDF file format."""
+
+import asdf
+import astropy
+
+
+def get_center_pixel(file, ra, dec):
+    """ Get the center pixel from a roman 2d science image """
+
+    # Get the roman 2D science image
+    with asdf.open(file) as f:
+        # Get the WCS
+        wcs = f['roman']['meta']['wcs']
+
+        # Get the WCS header
+        header = wcs.to_fits_sip()
+
+    # Update WCS header with some keywords that it's missing.
+    # Otherwise, it won't work with astropy.wcs tools (TODO: Figure out why. What are these keywords for?)
+    for k in ['cpdis1', 'cpdis2', 'det2im1', 'det2im2', 'sip']:
```

Comment:
are these always going to be missing? should we check first?

Reply:
I think these are related to handling image distortions.  They might not be common but it's not guaranteed these are always missing.  Adding a check makes sense so we don't accidentally overwrite.